activesupport/lib/active_support/json/encoding.rb

```diff
@@ -18,8 +18,12 @@ module JSON
     #
     #   ActiveSupport::JSON.encode({ team: 'rails', players: '36' })
     #   # => "{\"team\":\"rails\",\"players\":\"36\"}"
-    def self.encode(value, options = nil)
-      Encoding.json_encoder.new(options).encode(value)
+
+    class << self
+      def encode(value, options = nil)
+        Encoding.json_encoder.new(options).encode(value)
+      end
+      alias_method :dump, :encode
```

Comment:
I like the alias method approach here, I vote we got with this one for anything under `ActiveSupport`